src/redux/FruitSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { getFruit } from './apiCall';
+
+export const initialState = {
+  data: [],
+};
+const Fruit = createSlice({
+  name: 'fruit',
+  // initialState: {
+  //   data: [],
+  // },
```

Comment:
- [ ] Kindly remove commented-out code to keep your code base clean

Reply:
Sure